wordlistctl.py

```diff
@@ -245,224 +242,235 @@ def lst_func(parser: argparse.ArgumentParser) -> None:
+    parser = ArgumentParser(prog=f"{__project__}", description=f"{__description__}", argument_default=argparse.SUPPRESS,
+                            allow_abbrev=False, add_help=False)
 
-	# Add options
-	parser.add_argument("-v", "--version", action="version", type=str, 
-						version=f"{__project__} {__version__}")
-	parser.add_argument("-h", "--help", action="help", help="Display this message")
+    # Add options
+    parser.add_argument("-v", "--version", action="version", type=str, version=f"{__project__} {__version__}")
+    parser.add_argument("-h", "--help", action="help", help="Display this message")
 
-	subparsers = parser.add_subparsers()
+    subparser = parser.add_subparsers()
 
-	fetch = subparsers.add_parser("fetch", help="fetch wordlists")
-	fetch.add_argument("-l", "--wordlist", nargs="+", dest="wordlist",
-						help="Wordlist(s) to fetch")
-	fetch.add_argument("-d", "--decompress", action="store_true",
-                       help="decompress and remove archive")
-    fetch.add_argument("-w", "--workers", type=int, default=10,
-                       help="download workers [default: %(default)s]")
+    fetch = subparser.add_parser("fetch", help="fetch wordlists")
+    fetch.add_argument("-l", "--wordlist", nargs="+", dest="wordlist", help="Wordlist(s) to fetch")
+    fetch.add_argument("-d", "--decompress", action="store_true", help="decompress and remove archive")
+
+
+    fetch.add_argument("-w", "--workers", type=int, default=10, help="download workers [default: %(default)s]")
     fetch.add_argument("-u", "--useragent", default=f"{__project__}/{__version__}",
                        help="fetch user agent [default: %(default)s]")
     fetch.add_argument("-b", "--base-dir", default=f"{WORDLIST_PATH}", dest="basedir",
                        help="wordlists base directory [default: %(default)s]")
 
-	fetch.set_defaults(func=fetch_func)
+    fetch.set_defaults(func=fetch_func)
 
-	search = subparsers.add_parser("search", help="search wordlists")
+    search = subparser.add_parser("search", help="search wordlists")
     search.add_argument("search_term", help="what to search")
     search.add_argument("-l", "--local", action="store_true", default=False,
                         help="search local archives")
     search.add_argument("-b", "--base-dir", default=f"{WORDLIST_PATH}", dest="basedir",
                         help="wordlists base directory [default: %(default)s]")
     search.set_defaults(func=search_func)
 
-    lst = subparsers.add_parser("list", help="list wordlists")
-    lst.add_argument("-g", "--group",
-                     choices=["usernames", "passwords",
-                              "discovery", "fuzzing", "misc"],
-                     help="group")
-    lst.set_defaults(func=lst_func)
-
-	# Parse command line
-	results = parser.parse_args()
-	if sys.argv.__len__() == 1:
-		parser.print_help()
-		return
+    list = subparser.add_parser("list", help="list wordlists")
```

Comment:
`list` is an [inbuilt data-type](https://docs.python.org/3/tutorial/datastructures.html#more-on-lists). Do not override.

Reply:
fixed